test/metabase/query_processor_test.clj

```diff
@@ -154,7 +156,9 @@
      :id          {:special_type :type/PK
                    :base_type    (data/id-field-type)
                    :name         (data/format-name "id")
-                   :display_name "ID"}
+                   :display_name "ID"
+                   :min-value    nil
```

Comment:
Unfortunately we don't currently have anything that renames the keys automatically but if you add entries in https://github.com/metabase/metabase/blob/master/src/metabase/query_processor/annotate.clj#L188 that should handle renaming keys as appropriate for the query processor responses